webapp/package.json

```diff
@@ -7,6 +7,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "dotenv": "^11.0.0",
```

Comment:
You don't seem to be using this in this branch anymore, can you remove it?

It's good to avoid adding unnecessary dependencies in a PR in case it (somehow) broke something

Reply:
Done! One of the joys of pnpm uninstall 👌